src/auth/AuthContext.tsx

```diff
@@ -199,6 +203,10 @@ export const AuthContextProvider = ({children}: PropsWithChildren<{}>) => {
         authenticationType: mapAuthenticationType(state.user),
         signInWithCustomToken: useCallback(authSignInWithCustomToken, []),
         retryAuth,
+        debug: {
+          user: state.user,
+          idTokenResult: state.idToken,
```

Comment:
Needs updating after the [renaming](https://github.com/AtB-AS/mittatb-app/pull/4740#discussion_r1795142979)

Reply:
Interesting that GH haven't marked this as merge conflict after the other PR was merged. I'm curios, so I'll just merge it and see what happens 🤠

Reply:
Well, that didn't work 😅 You can do a retroactive PR on the master branch fix by giving thumbs up to this comment.
https://github.com/AtB-AS/mittatb-app/commit/00f52998c5744dc6600c90be75d41d73a6990680

Reply:
Shouldn't the lint check have failed?
We recently had a case with https://github.com/AtB-AS/mittatb-app/pull/4717 and https://github.com/AtB-AS/mittatb-app/pull/4714 being merged without the PR checks re-running, so a fix was required after: https://github.com/AtB-AS/mittatb-app/pull/4718.

Reply:
I don't think so, since the lint check is based on the current branch and don't know what has happened on the master branch.